internal/handlers/pg/msg_update.go

```diff
@@ -64,6 +65,11 @@ func (h *Handler) MsgUpdate(ctx context.Context, msg *wire.OpMsg) (*wire.OpMsg,
 
 	created, err := h.pgPool.CreateTableIfNotExist(ctx, sp.DB, sp.Collection)
 	if err != nil {
+		if errors.Is(pgdb.ErrInvalidTableName, err) ||
```

Comment:
If I understand correctly, in some similar cases, we use `switch - case` instead of `if`s. Peresonally, I'm ok with `if`s as well, so I leave it up to you to keep it as is or rewrite it to switch-case.

Reply:
> If I understand correctly, in some similar cases, we use `switch - case` instead of `if`s. Peresonally, I'm ok with `if`s as well, so I leave it up to you to keep it as is or rewrite it to switch-case.

well, let's use the following workflow:
1. we all agree on what to use, switch case or if-s
2. record it in our issue or somewhere else
3. I change.

For now, I'll leave it as it is.